packages/uswds-core/src/styles/_notifications.scss

```diff
@@ -18,9 +18,26 @@ $uswds-notifications:
 + "\A \2709  USWDS Notifications"
 + "\A --------------------------------------------------------------------"
 + "\A 3.0.0"
-+ "\A We deprecated the $output-all-utilities setting."
-+ "\A Control utility output with $output-these-utilities."
-+ "\A See designsystem.digital.gov/documentation/settings for more.";
++ "\A - Settings: We deprecated the $output-all-utilities setting."
++ "\A   Control utility output with $output-these-utilities."
++ "\A   See designsystem.digital.gov/documentation/settings for more."
++ "\A "
++ "\A --------------------------------------------------------------------"
++ "\A 3.1.0"
++ "\A - Accessibility: We added `type='button'` to all buttons that"
++ "\A   do not submit a form. All your non-form project buttons will"
++ "\A   need `type='button'`."
++ "\A - Accessibility: We updated the markup in the usa-password package"
++ "\A   to use a <button> element instead of an anchor element."
++ "\A - Accessibility: Elements that get `aria-disabled` instead of"
++ "\A    `disabled` are now styled as disabled."
++ "\A - Compatibility: We're using some CSS selectors not supported"
++ "\A   by IE11."
++ "\A - Display: We removed the outline circle from social media icons."
++ "\A - Settings: We changed the default value of $theme-hero-image."
++ "\A   Teams that use the default image (hero.png) will need to get the"
++ "\A   new image (hero.jpg) from the distribution package."
++ "\A ";
```

Comment:
We moved `@mixin embed-container` to `deprecated.scss` in the aspect-ratio PR. Not sure that it would warrant a notification, but wanted to share just in case.

https://github.com/uswds/uswds/pull/4811/files#diff-c2c5b5639aae3863e94643a3a613d89272dd926b077aae1c743a0eecabb9d355

Reply:
I've pushed a change adding this deprecation notice